v21.1/cockroach-auth-session.md

```diff
@@ -0,0 +1,208 @@
+~~~
+
+List all authenticated sessions to the HTTP interface, including currently active and recently expired sessions:
+
+~~~ shell
+$ cockroach auth-session list [flags]
+~~~
+
+View help:
+
+~~~ shell
+$ cockroach auth-session --help
+~~~
+
+~~~ shell
+$ cockroach auth-session {subcommand} --help
+~~~
+
+## Flags
+
+All three `auth-session` subcommands accept the standard [SQL command-line flags](cockroach-start.html#flags).
+
+In addition, the `auth-session login` subcommand supports the following flags.
+
+Flag | Description
+-----|------------
+`--expire-after` | Duration of the newly-created HTTP authentication token, after which the token expires. Specify the duration in numeric values suffixed by one or more of `h`, `m`, and `s` to indicate hour, minute, and second duration. See the [example](#log-in-to-the-http-interface-with-a-custom-expiry).<br><br>**Default:** `1h0m0s` (1 hour)
+`--only-cookie` | Limits output to only the newly-created HTTP authentication token (the "cookie") in the response, appropriate for output to other commands. See the [example](#log-in-to-the-http-interface-with-limited-command-output).
+
+## Response
+
+The `cockroach auth-session` subcommands return the following fields.
+
+### `auth-session login`
+
+Field | Description
+------|------------
+`username` | The username of the user authenticated.
+`session ID` | The session ID to the HTTP interface previously established for that user.
+`authentication cookie` | The cookie that may be used from the command line, or from other tools, to authenticate access to the HTTP interface for that user.
+
+### `auth-session logout`
+
+Field | Description
+------|------------
+`username` | The username of the user whose session was revoked.
+`session ID` | The session ID to the HTTP interface previously established for that user.
+`revoked` | The date and time of revocation for that user's authenticated session.
+
+### `auth-session list`
+
+Field | Description
+------|------------
+`username` | The username of the user authenticated.
+`session ID` | The session ID to the HTTP interface established for that user.
+`created` | The date and time a session was created.
+`expired` | The date and time a session expired.
+`revoked` | The date and time of revocation for that user's authenticated session. If the session is still active, this will appear as `NULL`.
+`last used` | The date and time of the last access to the HTTP interface using this session token.
+
+## Required roles
+
+To run any of the `auth-session` subcommands, you must be a member of the [`admin` role](authorization.html#admin-role). The user being authenticated via `login` or `logout` does not require any special roles.
+
+## Considerations
+
+- The `login` subcommand allows administrative users to create HTTP authentication tokens with an arbitrary expiration date and time. If operational policy requires stricter control of authentication sessions, you can:
```

Comment:
I'd just say `admin` in codes rather than "administrative", referring to the admin role as specified earlier in the doc. More exact as well as a shorter word. This applies in all cases throughout the PR.
In your example, looks like you specify duration, not expiration date/time.
Also, I suggest breaking things out a bit.
```suggestion
- The `login` subcommand allows `admin` users to create HTTP authentication tokens with arbitrary duration.
    
    {{site.data.alerts.callout_info}}
    If operational policy requires stricter control of authentication sessions, you can:
```